src/notification/index.js

```diff
@@ -265,29 +259,32 @@ export class NotificationListener {
   }
 }
 
-/** Try to cause a `remoteNotificationsRegistered` event. */
+/**
+ * Try to cause a `remoteNotificationsRegistered` event.
+ *
+ * Our 'register' listener will fire on that event.
+ */
 export const getNotificationToken = () => {
   if (Platform.OS === 'ios') {
-    // This leads to a call (in wix's, or RN upstream's, NotificationsIOS) to this:
+    // This leads to a call in RNC's PushNotificationIOS to this:
     //   https://developer.apple.com/documentation/uikit/uiapplication/1622932-registerusernotificationsettings
     // (deprecated after iOS 10, yikes!); which after possibly prompting the
-    // user causes "the app" (i.e. the platform part) to call this, I think,
-    // though I haven't successfully traced all the steps there:
```

Comment:
Does this mean you have traced through those steps? :slightly_smiling_face:

Reply:
I did, but I don't remember offhand what they all were, and there's a chance that I missed something—I'll come back to this and write up the steps.

Reply:
In my current revision, these comments contain more details, which I'm more sure about.